AGENTS.md

```
@@ -82,6 +86,24 @@ src/scripts/         scripts sueltos, cada uno con su entorno PEP 723
 .cursor/rules/       punteros para Cursor a las skills de arriba
 ```
 
+## Idioma
```

markdownlint-cli2 (0.23.2)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)